presentation/src/main/java/com/keeply/presentation/core/components/AppBar.kt

```diff
@@ -0,0 +1,261 @@
+package com.keeply.presentation.core.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.keeply.presentation.core.theme.KeeplyTheme
+
+@Composable
+fun KeeplyAppBar(
+    modifier: Modifier = Modifier,
+    title: String? = null,
+    leadingIcon: @Composable (() -> Unit)? = {}, // null 이면 아이콘 영역이 할당되지 않음
+    trailingIcon: @Composable (() -> Unit)? = {}, // null 이면 아이콘 영역이 할당되지 않음
+    trailingText: String? = null,
+    customTitleContent: @Composable (BoxScope.() -> Unit)? = null,
+    onClickLeading: (() -> Unit)? = null,
+    onClickTrailing: (() -> Unit)? = null,
+    backgroundColor: Color = KeeplyTheme.colors.neutral100,
+    contentColor: Color = KeeplyTheme.colors.neutralBlack,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(60.dp)
+            .background(backgroundColor)
+            .padding(start = 16.dp, end = 16.dp, top = 12.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        leadingIcon?.let {
+            AppBarIconButton(
+                content = leadingIcon,
+                enabled = onClickLeading != null,
+                onClick = { onClickLeading?.let { it() } }
+            )
+        }
+
+        Box(
+            modifier = Modifier
+                .weight(1f),
+            contentAlignment = Alignment.Center
+        ) {
+            title?.let {
+                KeeplyText(
+                    text = it,
+                    style = KeeplyTheme.typography.subtitle01,
+                    color = contentColor,
+                    textAlign = TextAlign.Center
+                )
+            }
+
+            customTitleContent?.let { it() }
+        }
+
+        if (trailingText.isNullOrEmpty().not()) {
+            KeeplyText(
+                text = trailingText,
```

Comment:
p3: null 일때 default값 추가해주면 좋을 것 같습니다~
```suggestion
                text = trailingText ?: "",
```